src/utils/helper.ts

```diff
@@ -108,6 +108,10 @@ export function isDict(v) {
     return typeof v==='object' && v!==null && !(v instanceof Array) && !(v instanceof Date);
 }
 
+export function ToJsonString(_in: any): string {
+    return JSON.stringify(_in).split(/[\r\n]+/).join("").split("\\").join("\\\\").split("'").join("\\'")
```

Comment:
Maybe you should use EscapeString here? I remember there's a case of string contains $ in it, but seems without \ in front of $. then it does not work.